bin.js

```diff
@@ -26,9 +26,9 @@ var argv = minimist(process.argv.splice(2), {
 })
 
 function usage () {
-  console.log('Usage: pes-client SCHEMA [--port PORT] [--host HOST]\n' +
-              '                  [--target-host HOST] [--target-port PORT]\n' +
-              '                  [--db PATH] [--no-repl] [--from-scratch]')
+  console.log('Usage: hypem <SCHEMA> [--schema SCHEMA] [--port PORT] [--host HOST]\n' +
```

Comment:
Maybe we just need a bigger help. See https://github.com/mcollina/help-me and https://github.com/mcollina/commist